Protoscope/ConsoleView.swift

```diff
@@ -40,7 +40,7 @@ class ConsoleView: UIView {
 		textView.textStorage.deleteCharactersInRange(NSMakeRange(0, textView.textStorage.length))
 	}
 
-	override init() {
```

Comment:
This is so weird. I had to fix it for [iOS too](https://github.com/Khan/iOS/commit/ee88c312d397bf995de108db1c5a744b932232f9). Why wouldn't this require `override` anymore?